packages/babel-core/test/config-chain.js

```diff
@@ -533,14 +536,22 @@ describe("buildConfigChain", function () {
     );
   });
 
-  it("json-config-error", function () {
+  it("json-config-error", function() {
     assert.throws(
-      function () {
+      function() {
         buildConfigChain({
           filename: fixture("json-config-error", "src.js"),
         });
       },
       /Error while parsing JSON/
     );
   });
+
+  it("env-deprecated", function() {
+    buildConfigChain({
+      filename: fixture("env-deprecated", "src.js"),
+    });
+
+    assert(console.log.calledWithMatch(/^The "env" option has been deprecated/));
```

Comment:
So, I'm a bit confused by this. When I stub `console.log` in this file it errors saying that it's already been stubbed. This also happens when I skip the other file that I stub `console.log`, so I think I'm cleaning up after myself correctly. Anyone else see/know what might be causing this?